RecoBTag/TensorFlow/plugins/sorting_modules.cc

```diff
@@ -1,5 +1,5 @@
 
```

Comment:
this should have been moved to src/, now that all other tools were mover to src/ and exposed in interface/